src/Cars.css

```diff
@@ -0,0 +1,172 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter+Tight&display=swap');
+
+body {
+  font-family: 'Inter Tight', sans-serif;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.car-card {
+  width: 300px;
+  height: 300px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  margin: 1em;
+  box-shadow: 1px 1px 8px 1px rgba(0, 0, 0, 0.2);
+  transition-property: transform;
+  transition-duration: 0.1s;
+  transition-timing-function: ease-in;
+  padding: 2em;
+  border-radius: 0.5em;
+}
+
+.car-card:hover {
+  transform: scale(1.025);
+  cursor: pointer;
+  box-shadow: 1px 1px 8px 1px rgba(22, 65, 255, 0.3);
+  border: 1px solid #060b26;
+}
+
+.main-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+  height: 100vh;
+
+  /* border: 2px solid red; */
```

Comment:
- [ ] Please, kindly remove this old piece of commented code in order to improve the cleanliness of the repo 💪